src/trip.c

```diff
@@ -284,3 +285,59 @@ Trips* GetTrips(CURL* curl, const char* origin, const char* destination) {
 
   return trips;
 }
+
+void CoordinatesForStations(CURL* const curl) {
+  // This is a reference to a journey detail. More logic need to be implemented
+  const char* k_ref =
```

Comment:
@TexWO There is no need to hardcode this value, since it's available in the `trips->trips` array in `application.c`, i.e. it should be dynamic.